index.html

```diff
@@ -1 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Lato:wght@700;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/dcdd89c74c.js" crossorigin="anonymous"></script>
+    <script src="js/nav.js" defer></script>
+    <script src="js/programs.js" defer></script>
+    <script src="js/popup.js" defer></script>
+    <script src="js/extra.js" defer></script>
+    <title>De Royale Institute</title>
+</head>
+<body>
+    <!-- =======================================
+                Nav Bar
+    ======================================== -->
+    <section class="socials mob-none">
```

Comment:
Kindly remove the border occurring when hovering. Instead, use only orange color as you did on About link. And please use the orange border only for the last element on the nav which is "De Royale" on your nav.
<img width="1251" alt="Screenshot 2023-01-04 at 3 01 23 AM" src="https://user-images.githubusercontent.com/25887253/210461340-908da205-96bb-4e40-a370-27b44bec80c6.png">
<img width="749" alt="Screenshot 2023-01-04 at 3 01 47 AM" src="https://user-images.githubusercontent.com/25887253/210461342-68648ba6-b4bc-4d57-b58e-1bb9b46af8a5.png">